### PR TITLE
Category published

### PR DESCRIPTION
**Pull request by fefrain**

### CODE REVIEW

File: server/modules/category/repository/categoryRepository.js [hunk 1]
```diff
@@ -194,6 +206,12 @@ const updateCategoryValidation = async (surveyId, categoryUuid, validation, clie
     [validation, categoryUuid]
   )
 
+const markCategoriesPublished = async (surveyId, client = db) =>
```
Comment (minotogna) [inline server/modules/category/repository/categoryRepository.js]: I would call this method ```markCategoriesPublishedBySurveyId``` what do you think?

File: server/modules/category/repository/categoryRepository.js [hunk 2]
```diff
@@ -107,6 +108,17 @@ const fetchCategoryAndLevelsByUuid = async (surveyId, categoryUuid, draft = fals
   return R.pipe(R.values, R.head)(categories)
 }
 
+const isCategoryPublished = async (surveyId, categoryUuid, client = db) => {
```
Comment (minotogna) [inline server/modules/category/repository/categoryRepository.js]: I wouldn't create a method just to check if it's published. you can use fetchCategoryAndLevlesByCategoryUuid and use Category.isPublished.

File: server/modules/category/repository/categoryRepository.js (hunk 1 above)
Comment (fefrain) [reply server/modules/category/repository/categoryRepository.js]: Is there any other markCategoriesPublished we should distinguish from?

File: server/modules/category/repository/categoryRepository.js (hunk 2 above)
Comment (fefrain) [reply server/modules/category/repository/categoryRepository.js]: In this way you avoid to fetch all the category, but I'll change it

Comment (minotogna) [reply server/modules/category/repository/categoryRepository.js]: in this way, if we decide to change how we handle a published category, you need to update the code twice: in the Category.isPublished and in this method. it's about consistency. Fetching 1 category with levels is a very fast operation

File: server/modules/category/repository/categoryRepository.js (hunk 1 above)
Comment (minotogna) [reply server/modules/category/repository/categoryRepository.js]: just to be consistent with other methods in the same module. whatever works in the end, your call

File: server/modules/category/repository/categoryRepository.js (hunk 2 above)
Comment (fefrain) [reply server/modules/category/repository/categoryRepository.js]: right, I'll change it

File: server/modules/category/repository/categoryRepository.js (hunk 1 above)
Comment (fefrain) [reply server/modules/category/repository/categoryRepository.js]: ok

**Pull request merged**